src/main/java/com/tfgm/TfgmApplication.java

```diff
@@ -0,0 +1,36 @@
+package com.tfgm;
+
+import com.tfgm.services.TramStopService;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.concurrent.TimeUnit;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.context.event.ApplicationReadyEvent;
+import org.springframework.context.event.EventListener;
+
+@SpringBootApplication
+public class TfgmApplication {
+
+  public static void main(String[] args)
+      throws URISyntaxException, IOException, InterruptedException {
+//    SpringApplication.run(TfgmApplication.class, args);
+
+      mainRunner();
+
+  }
+
+  @EventListener(ApplicationReadyEvent.class)
```

Comment:
Have a look at https://www.baeldung.com/spring-boot-configure-multiple-datasources for configuring an existing API as your datasource, this should let the application run as expected